DiscordBot/Extensions/UserDBRepository.cs

```diff
@@ -11,11 +11,8 @@ public class ServerUser
         // ReSharper disable once InconsistentNaming
         public int ID { get; private set; }
         public string Username { get; set; }
-        public string Discriminator { get; set; }
         // ReSharper disable once InconsistentNaming
         public string UserID { get; set; }
-        public string Avatar { get; set; }
-        public string AvatarUrl { get; set; }
         public DateTime JoinDate { get; set; }
```

Comment:
We can get JoinDate through the API: https://docs.stillu.cc/api/Discord.WebSocket.SocketGuildUser.html#Discord_WebSocket_SocketGuildUser_JoinedAt
Since the Fire Reset, the database doesn't include anymore the date from UDH, so I think there's no benefit in storing it?

The only difference that will make is:
- With our current system, we store the earliest join date, so quitting and re-joining doesn't update.
- With the API it's only the date of the last join.

IMO we don't care, and any other bot will show the API one anyway.